src/media/js/views/search.js

```diff
@@ -117,6 +117,7 @@ define('views/search',
         };
 
         var query = $('#search-q').val() || $('#search-q-desktop').val();
+        var query = query.replace( /  +/g, ' ' );
```

Comment:
Sorry for the delay @ddurst .

The regex `/\s\s+/g` doesn't catch single non white space strings.  The one you suggested `/\s\s*/g` does a perfect job. I checked it in an online regex match tool and it works as required.

If this regex is fine, I'll make the changes and add a commit it. :)